stdlib/public/core/Process.swift

```diff
@@ -56,10 +63,19 @@ public enum Process {
   /// around by the optimizer which will break the data dependence on argc
   /// and argv.
   public static var arguments: [String] {
-    if _arguments == nil { 
-      _arguments = _computeArguments()
+    let argumentsPtr = UnsafeMutablePointer<AnyObject?>(
+                       Builtin.addressof(&_swift_stdlib_ProcessArguments))
+
+    // Check whether argument has been initialized.
+    if let arguments = _stdlib_atomicLoadARCRef(object: argumentsPtr) {
+      return (arguments as! StringArgument).args
     }
-    return _arguments!
+
+    // Initialize argument.
```

Comment:
Please drop the two comments above -- they are completely redundant with the implementation.